src/test/java/bio/terra/service/rawls/RawlsServiceTest.java

```diff
@@ -48,20 +40,20 @@ public class RawlsServiceTest {
   private static final String OWNER_EMAIL = "policy-email@firecloud.org";
   private static final String WORKSPACE_NAME = "testWorkspace";
   private static final String WORKSPACE_NAMESPACE = "testNamespace";
+  private static final String WORKSPACE_LINK =
+      "%s/#workspaces/%s/%s".formatted(BASE_PATH, WORKSPACE_NAMESPACE, WORKSPACE_NAME);
```

Comment:
It's a little awkward that this duplicates the code in `getWorkspaceLink()`. But I guess it's not all that much code that's being duplicated here.

Reply:
I'm with you, I spent too much time going back and forth on this. I guess this is a place where a `@VisibleForTesting` annotation is appropriate, I'll use that instead.